javalib/src/main/scala/java/time/LocalTime.scala

```diff
@@ -0,0 +1,377 @@
+package java.time
+
+import scala.scalajs.js
+
+import java.time.temporal._
+
+final class LocalTime private (hour: Int, minute: Int, second: Int, nano: Int)
+    extends Temporal with TemporalAdjuster with Comparable[LocalTime]
+    with java.io.Serializable with Preconditions {
```

Comment:
You can't change the inheritance definitions of a class ported from java. You need to remove `with Preconditions`, but you could redefine it as `object Preconditions {...}` and the import its method with `import Preconditions.require`.